packages/xlsx2csv/xlsx2csv.0.1.2/opam

```diff
@@ -0,0 +1,42 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Simple library to export .xlsx files as CSVs"
+description: "Simple library to export .xlsx files as CSVs"
+maintainer: ["leomendesantos@gmail.com"]
+authors: ["Leonardo Santos"]
+license: "GPLv3"
```

Comment:
```suggestion
license: "GPL-3.0-only"
```
if I am not mistaken, can you double check?

Reply:
That seems to have fixed it, thanks!